apps/src/lab2/lab2Redux.ts

```diff
@@ -53,13 +53,16 @@ export interface LabState {
   // Initial sources for the current level, as loaded from the server. Subsequent changes to sources
   // while the project is being edited are managed by the Lab and Project Manager directly.
   initialSources: ProjectSources | undefined;
-  // Level data for the current level
-  levelData: LevelData | undefined;
-  hideShareAndRemix: boolean;
-  isProjectLevel: boolean;
   // Validation status for the current level. This is used by the progress system to determine
   // what instructions to display and if the user has satisfied the validation conditions, if present.
   validationState: ValidationState;
+  // Level properties for the current level.
+  levelProperties: LevelProperties | undefined;
+  // Note: the following properties are derived from levelProperties. We should update components that consume
```

Comment:
And when components are updated to use `levelProperties`,  the fields below can be removed?

Reply:
Yep!